app/app.css

```diff
@@ -94,3 +105,94 @@
   font-size: 1.2vmin;
   word-wrap: break-word;
 }
+
+.bookDetail{
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  margin: 30px 0 30px 0;
+}
+
+.bookDetail__img{
+  width: 191px;
+  height: 272px;
+  background-color: "#fafafa";
+  margin-right: 40px;
+}
+
+.bookRecomendations__img{
+  width: 87px;
+  height: 124px;
+  background-color: "#fafafa";
+  margin-right: 40px;
+}
+
+.bookDetail__img img ,.bookRecomendations__img img{
+  width: 100%;
+  height: 100%;
+}
+
+.bookDetail__content{
+  height: 272px;
+  display: flex;
+  flex-direction: column;
+  align-content: flex-start;
+}
+
+.bookDetail__content p {
```

Comment:
You could probably use a named class here, it makes the style more understandable. For example `.bookDetail__description`, I really don't know what the `p` is by just reading this style.

Reply:
It actually exists, it's `bookDetail__attr`